plugins/database/postgresql/postgresql.go

```diff
@@ -48,7 +48,13 @@ var (
 	singleQuotedPhrases = regexp.MustCompile(`('.*?')`)
 )
 
-// New implements builtinplugins.BuiltinFactory
+func NewWithMultiplex() (dbplugin.Database, error) {
```

Comment:
I wonder if `NewWithMultiplex` is needed since it's pretty much the same as `New`. From the plugin's perspective, all they might to do to enable multiplexing is for their main func to call `ServeMultiplex` with the factory func provided (which is what you're doing in `postgresql-database-plugin/main.go`).

I noticed that we do something that's pretty similar already on today's non-DB plugins: https://github.com/hashicorp/vault/blob/a24653cc5cca475962b27b71dcba29a4aa94d6ee/sdk/plugin/serve.go#L22-L30

Reply:
addressed in https://github.com/hashicorp/vault/pull/13734/commits/e093c883f89a49d98395c796eeb9416961c33306